var/spack/repos/builtin/packages/cray-libsci/package.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from llnl.util.filesystem import LibraryList
+from spack import *
+
+
+class CrayLibsci(Package):
+    """The Cray Scientific Libraries package, LibSci, is a collection of
+    numerical routines optimized for best performance on Cray systems."""
+
+    homepage = "http://www.nersc.gov/users/software/programming-libraries/math-libraries/libsci/"
+    url      = "http://www.nersc.gov/users/software/programming-libraries/math-libraries/libsci/"
```

Comment:
I would remove this URL if you don't plan on fetching any versions.

Reply:
Having no `url` breaks things.

Reply:
@tgamblin @tldahlgren I thought we fixed this bug.